assignments/assignment02.rb

```diff
@@ -10,9 +10,27 @@
 # creates an english string from array
 
 def to_sentence(ary)
-  # your implementation here
+  sentence = ""
+  if ary.length <= 1
+    sentence = ary.join
+  else
+    ary.insert(ary.length-1, 'and')
+    c = 0
+    while c <= ary.length-1
```

Comment:
this is too complicated. Think of using join(", ") with the appropriate subset of the array.